Add a note to OpenCL (#6039)
qgis/QGIS-Documentation: docs/user_manual/introduction/qgis_configuration.rst

   Acceleration tab

> .. note:: depending on your machine you have to install additional libraries to
>   enable the OpenCL acceleration.

Reviewer: ```suggestion
.. note:: depending on your hardware and software, you may have to install additional
   libraries to enable OpenCL acceleration.
```
